packages/@lwc/integration-karma/test/light-dom/lifecycle/index.spec.js

```diff
@@ -0,0 +1,136 @@
+import { createElement } from 'lwc';
+import { extractDataIds, USE_LIGHT_DOM_SLOT_FORWARDING } from 'test-utils';
+
+import SlotForwarding from 'x/slotForwarding';
+import DynamicSlotForwarding from 'x/dynamicSlotForwarding';
+
+import { resetId } from './util.js';
+
+const resetTimingBuffer = () => {
+    window.timingBuffer = [];
+};
+
+if (USE_LIGHT_DOM_SLOT_FORWARDING) {
```

Comment:
Oh I see, this tests that slot forwarding works correctly.

I would add another, simpler test to confirm that `disconnectedCalback` fires both with and without slot forwarding. This will confirm the fix from this PR.

Reply:
Added a simple test for the standard slot scenario! - 998a496a90aaef02ba1942e1b75f3b5de4c25903

Reply:
Confirmed the fix – without your PR, the `standard slotting` test fails in the latest API version but passes in `API_VERSION=60`.